python/docs/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    = -n -v -W --keep-going
+SPHINXOPTS    = -n -v
```

Comment:
This change is temporary and will be removed before we merge (the current implementation adds a couple of warnings, but they will go away once we organize the C++ docs more properly).

Reply:
Removed.